terraform/instances.tf

```diff
@@ -62,7 +62,7 @@ resource "google_compute_instance" "sacrificial_vm" {
 resource "google_compute_instance" "logger_vm" {
   name         = "logger-vm"
   machine_type = var.machine_type
-  tags         = ["observer"]
+  tags         = ["logger"]
```

Comment:
might be easier to identify

Reply:
yes, it makes sense, sorry for my bad naming sense xD